packages/db/prisma/schema.prisma

```diff
@@ -43,7 +43,8 @@ model Activity {
 }
 
 model Absence {
-  id          Int      @id @default(autoincrement())
   userId      Int
-  absenceDate DateTime @db.Date
+  absenceDate DateTime
+
+  @@unique([absenceDate])
```

Comment:
this should also include the userId -- right now it just means that every absence date should be unique, but we want the user + date combo to be unique

Reply:
oh right tyty